clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -388,14 +388,15 @@ def LoadOp : CIR_Op<"load", [
   }];
 
   let arguments = (ins Arg<CIR_PointerType, "the address to load from",
-                           [MemRead]>:$addr, UnitAttr:$isDeref);
+                           [MemRead]>:$addr, UnitAttr:$isDeref,
+                       UnitAttr:$is_volatile);
   let results = (outs CIR_AnyType:$result);
 
   // FIXME: we should not be printing `cir.ptr` below, that should come
   // from the pointer type directly.
   let assemblyFormat = [{
-    (`deref` $isDeref^)? $addr `:` `cir.ptr` type($addr) `,`
-                                   type($result) attr-dict
+    (`deref` $isDeref^)? (`volatile` $is_volatile^)?
+    $addr `:` `cir.ptr` type($addr) `,` type($result) attr-dict
```

Comment:
Please also update the docs for each changed Op to mention about volatile.

Reply:
Updated.